File: common/src/main/java/work/fking/pangya/common/model/server/ServerBoost.java
package work.fking.pangya.common.model.server;

public enum ServerBoost {
    DOUBLE_PANG(1 << 1),
    DOUBLE_EXP(1 << 2),
    ANGEL_EVENT(1 << 3),
    TRIPLE_EXP(1 << 4),
    CLUB_MASTERY(1 << 7);

    private final int value;

    ServerBoost(int value) {
        this.value = value;
    }

    public int value() {
        return value;
    }
}

File: common/src/main/kotlin/work/fking/pangya/common/server/ServerBoost.kt
package work.fking.pangya.common.server

enum class ServerBoost(
    val value: Int
) {
    DOUBLE_PANG(1 shl 1),
    DOUBLE_EXP(1 shl 2),
    ANGEL_EVENT(1 shl 3),
    TRIPLE_EXP(1 shl 4),
    CLUB_MASTERY(1 shl 7);
}
